pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -51,15 +52,19 @@ class ModificationsExportDialog extends React.Component {
     }
 
     handleSelect(event, active_tab) {
-        this.setState({ active_tab });
+        this.setState({ active_tab, copied: false });
+        if (this.timeoutId !== null) {
+            clearTimeout(this.timeoutId);
+            this.timeoutId = null;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19069-20230831-104711-4f88c3d0-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)